playwright/e2e/crypto/utils.ts

```diff
@@ -377,3 +377,14 @@ export async function awaitVerifier(
         return verificationRequest.verifier;
     });
 }
+
+/** Log in a second device for the given bot user */
+export async function createSecondBotDevice(page: Page, homeserver: HomeserverInstance, bob: Bot) {
```

Comment:
Playwright allow to add [fixture](https://playwright.dev/docs/test-fixtures) which allows to encapsulate and easily share behaviour across the tests.
You can take a look at `pinned-messages.spec.ts` to see an example.

Reply:
Frankly, I don't find our use of fixtures very helpful, and think we should use them less rather than more. They are too "magical", and not flexible enough, whilst a separate function is very explicit, and not much more code.

In this instance: there is a bunch of preparation which has to happen before the second device is created. I don't think I can do that with a fixture?